session-02-big-picture/README.md

```diff
@@ -257,6 +257,43 @@ Possibly prerecorded videos. Need offline support
        ```git clone https://github.com/koushikkothagal/java-backend-bootcamp.git```
     
 
+## On Linux
+
+Linux comes with git pre-installed, you can check the version of git by typing the following command in terminal
+
+```console
+$ git --version
+git version 2.37.1
+```
+
+If that's not the case, follow the steps below to install git
+
+1. Open terminal and type the following command to update the package list
+
+```console
+sudo apt-get update
+```
+2. Type the following command to install git
+```console
+sudo apt-get install git
+```
+![git installing](./assets/git-linux-setup/installing-git.png)
```

Comment:
```suggestion
![git installing](../assets/git-linux-setup/installing-git.png)
```

Reply:
assets exist at the same level as readme, I don't think, this needs to go one step back

Reply:
Ah, my bad then. Looks good!